packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -99,9 +107,10 @@ function createIframeVirtualEnvironment(
         blueCreateHooksCallbackCache.set(blueRefs.document, blueConnector);
     }
     const { eval: redIndirectEval } = redWindow;
+    const signedConnectorEval = (v: string) => redIndirectEval(trustedCreateScript(v));
     const env = new VirtualEnvironment({
         blueConnector,
-        redConnector: createRedConnector(redIndirectEval),
+        redConnector: createRedConnector(signedConnectorEval),
```

Comment:
Did we want to call the callback something like `signSourceText` or even `sourceTextCallback`? I'm fine with the "trusted" term but in the dev sync it sounded like there wanted to be some distance from it.

Reply:
👍